### PR TITLE
Docs - Add section about how HTTP maps to gRPC

### PR DESCRIPTION
**Pull request by philipithomas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,14 @@ But patch is welcome.
 * Encoding request/response body in XML
 * True bi-directional streaming. (Probably impossible?)
 
+# Mapping gRPC to HTTP
+
+* [gRPC to HTTP header code mappings](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/errors.go#L15)
```
Comment (achew22) [inline README.md]: It isn't really a "header code" it is a http status code.

File: README.md [hunk 2]
```diff
@@ -220,6 +220,14 @@ But patch is welcome.
...
+* HTTP request source IP is added as `X-Forwarded-For` gRPC metadata
```
Comment (achew22) [inline README.md]: It is added as a gRPC header

**Comment by achew22** [comment]
This looks good to me. @yugui do you know what's up with the cla bot?

**Comment by philipithomas** [comment]
@achew22 I have a CLA on file, if you can verify through my Github username.

**Comment by yugui** [comment]
LGTM. Thank you for your contribution.

**Pull request merged**